meet-data/oceaniapf/1401/entries.csv

```diff
@@ -432,4 +434,4 @@ David Jame,AUS,M-O,104.35,105,-320,-350.5,-350.5,,170,-177.5,-177.5,170,280,-300
 Denken Tom,NAR,M-J,113.05,120,,,,,-190,-190,-190,,,,,,,M,SBD,Single-ply,DQ
 James Tsiode,NAR,M-O,119.4,120,240,260,280,280,-240,250,270,270,200,220,240,240,790,M,SBD,Single-ply,1
 Jesse Jeremiah,NAR,M-O,150.8,120+,-330,-330,350,350,220,240,,240,220,250,-280,250,840,M,SBD,Single-ply,1
-Tom Martin,UK,M-O,93.2,105,255,265,-275,265,162.5,170,-172.5,170,330,-343,-343,330,765,M,SBD,Raw,G
+Tom Martin,UK,M-O,93.2,105,255,265,-275,265,162.5,170,-172.5,170,330,-343,-343,330,765,M,SBD,Raw,2
```

Comment:
The places in this file shouldn't be changed -- Tom Martin was accidentally counted in the `M-O` instead of as a Guest.